File: benchmark/corpus/04_off_by_one/a_sum_inclusive.js
// Sums a list with "<=" where "<" was meant, reading one past the end.
function total(xs) {
  let sum = 0;
  let i = 0;
  while (i <= xs.length) {
    sum = sum + xs[i]; // DANGER
    i = i + 1;
  }
  return sum;
}

console.log(total([2500, 45000, 12000]));

File: benchmark/corpus/04_off_by_one/a_sum_inclusive.py
# Sums a list with "<=" where "<" was meant, reading one past the end.


def total(xs):
    s = 0
    i = 0
    while i <= len(xs):
        s = s + xs[i]  # DANGER
        i = i + 1
    return s


print(total([2500, 45000, 12000]))
